platform/macos/app/Base.lproj/MainMenu.xib

```diff
@@ -542,9 +542,9 @@
                                     <action selector="dropManyPins:" target="-1" id="Rtv-8N-3Z8"/>
                                 </connections>
                             </menuItem>
-                            <menuItem title="Add Polygon and Polyline" keyEquivalent="l" id="DVr-vT-lpe">
+                            <menuItem title="Simulate Nightfall" keyEquivalent="l" id="DVr-vT-lpe">
```

Comment:
nit: Can you add a subtitle/tooltip that this item uses Spherical caps or `MGLCircle`